### ISSUE TITLE
Backend frequency for EdgeDB cloud

### ISSUE THREAD

**Issue by janwirth** [edgedb/edgedb#7569]
We would like hourly / quarter-hourly backups for 48 hours. Our database dump is tiny as of now (<2mb)

**Comment by elprans** [comment]
Any specific reason you'd want frequent backups?  Is it a worry about backup recency?  EdgeDB instances are being archived continuously, i.e. aside from regular backup "snapshots" we ship WAL to durable storage, so you can always restore to a state within last 5 minutes or so.  We're also working on the ability to restare to an arbitrary point in time within the backup retention period.

**Comment by janwirth** [comment]
No, it just seems that the last backup available can be up to 24h old, so we might lose up to 24h of data. It seems that I was mistaken here.
